### PR TITLE
Add/update Firefox data for Window API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR updates the Firefox data for the Window API based upon manual testing to achieve more real data for the Window API.

Tests used: https://mdn-bcd-collector.appspot.com/tests/api/Window

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -8468,19 +8450,19 @@
             ],
             "firefox": [
               {
-                "version_added": true
+                "version_added": "1"
```
Comment (foolip) [inline api/Window.json:8453]: Is the "alternative_name" `pageXOffset` what you tested here? http://mdn-bcd-collector.appspot.com/tests/api/Window/pageXOffset exists, but our update scripts don't handle alternative names and prefixes correctly yet...

**Review by foolip — COMMENTED** [review]
Just a few minor changes needed on this one too. Good stuff, making progress!

File: api/Window.json [hunk 2]
```diff
@@ -2125,26 +2125,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "63",
```
[the remark this answers is not in the document]
Comment (vinyldarkscratch) [reply api/Window.json:2128]: Ack, you're right -- I was thinking Chrome version numbers for some odd reason...

File: api/Window.json [hunk 3]
```diff
@@ -4635,8 +4620,7 @@
               "version_removed": "79"
             },
             "firefox": {
-              "version_added": "22",
-              "version_removed": "60"
```
[the remark this answers is not in the document]
Comment (vinyldarkscratch) [reply api/Window.json:4639]: It looks like this data came from the initial MDN wiki migration, so I don't trust the accuracy of the version_removed.  Double-checked in Firefox 81 as well, and support is present.

File: api/Window.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/Window.json:8453]: I tested both `scrollX` and `pageXOffset` to make sure both of them were present!

File: api/Window.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (foolip) [reply api/Window.json:4639]: Thanks for confirming!

File: api/Window.json (hunk 1 above)
Comment (foolip) [reply api/Window.json:8479]: Thank you!

**Review by foolip — APPROVED** [review]

**Pull request merged**